content/en/docs/concepts/storage/storage-capacity.md

```diff
@@ -0,0 +1,116 @@
+---
+reviewers:
+- jsafrane
+- saad-ali
+- msau42
+- xing-yang
+- pohly
+title: Storage Capacity
+content_type: concept
+weight: 45
+---
+
+<!-- overview -->
+
+Storage capacity is limited and may vary depending on the node on
+which a pod runs: network-attached storage might not be accessible by
+all nodes, or storage is local to a node to begin with.
+
+This page describes how Kubernetes keeps track of storage capacity and
+how the scheduler uses that information to schedule Pods onto nodes
+that have access to enough storage capacity for the remaining missing
+volumes. Without storage capacity tracking, it is random whether the
+selected node can run the Pod and multiple scheduling retries may be
+needed.
+
```

Comment:
_Ideally_ I'd like to see `{{< glossary_tooltip text="CSI" term_id="csi" >}}` worked into the early part of this page, so that readers see an explanatory tooltip. `{{< glossary_tooltip text="Container Storage Interface" term_id="csi" >}} (CSI)` is fine too.

Reply:
I've added `Tracking storage capacity is supported for {{< glossary_tooltip
text="Container Storage Interface" term_id="csi" >}} (CSI) drivers.`

Reply:
I actually made it even more complex by making it a link to /docs/concepts/storage/volumes/#csi - let's see whether that renders correctly.

Reply:
Yes, it does (checked in the preview).